README.md

```diff
@@ -1,5 +1,11 @@
 # gem-plotting-tools
 
+Branch|[![Travis CI logo](TravisCI.png)](https://travis-ci.org)|[![Coveralls logo](Coveralls.png)](https://www.coveralls.io)|[![Codecov logo](Codecov.png)](https://www.codecov.io)|[![Codacy logo](Codacy.png)](https://www.codacy.com)|[![Landscape logo](Landscape.png)](https://www.landscape.io)|  [![CodeClimate logo](Codeclimate.png)](https://www.codeclimate.com)
```

Comment:
If I check on the rendered [README.md](https://github.com/jsturdy/gem-plotting-tools/blob/jsturdy-integrations-rebase/README.md) from your github page the images do not load.

Minor detail.

Reply:
Yes, this will be a post merge update as well (I did it once for one repo, and just have to make the syntax everywhere consistent)